subprojects/testing-base/src/integTest/groovy/org/gradle/testing/TestResultIntegrationTest.groovy

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I'm not sure that this is a good place to have this test. Since it's coupled to JUnitPlatform integration, probably it's better to have it in :testing-jvm project (see JUnitPlatformIntegrationSpec and inheritors).
WDYT @donat ?

Reply:
Also seems like having this test there will allows us to reuse existing setup for project-under-test of JUnitPlatformIntegrationSpec

Reply:
Sounds good to me.